global_body_planner:
  ros__parameters:
    body_plan: "body_plan"
    discrete_body_plan: "discrete_body_plan"
    param_str: "HelloYmalParams"
    update_rate: 1
    num_calls: 1            # Number of times to call the planner
    replan_time_limit: 1.0  # Minimum amount of time to run the planner (will replan to find better solutions until this duration is up)
    algorithm: rrt-connect  # rrt-connect or rrt-star-connect

terrain_map_publisher:
  ros__parameters:
    terrain_map: "terrain_map"
    update_rate: 10
    map_data_source: csv
    terrain_type: slope     # slope or rough_terrain


